.gitignore

```diff
@@ -0,0 +1 @@
+.idea/
```

Comment:
can we add a .gitignore too account for local values?

Reply:
what do you mean?

Reply:
this IS the .gitignore